manifests/j/JanProchazka/dbgate/5.2.6/JanProchazka.dbgate.installer.yaml

```diff
@@ -15,11 +15,8 @@ Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/dbgate/dbgate/releases/download/v5.2.6/dbgate-5.2.6-win_x64.exe
   InstallerSha256: D98C843AC993D9B43DBFB4A06B4487D0937C9C0AD2E4041F45D2021BAC1873FB
-- Architecture: x86
-  InstallerUrl: https://github.com/dbgate/dbgate/releases/download/v5.2.6/dbgate-5.2.6-win.exe
-  InstallerSha256: e124565d0e9788f75e89e75ce14b270cdaf7c679ceaba043a570ecbe17fcddc5
 - Architecture: arm64
-  InstallerUrl: https://github.com/dbgate/dbgate/releases/download/v5.2.6/dbgate-5.2.6-win_arm64.exe
-  InstallerSha256: f758737de4b37c9a972ccb04429d875261321ccc1dd6ec3917190d87dd328547
+  InstallerUrl: https://github.com/dbgate/dbgate/releases/download/v5.2.6/dbgate-5.2.6-win_x64.exe

```

Comment:
@stephengillie - This might be an issue too;

TBH, I don’t think there is *no* x86 installer, rather just that the URLs were mis-matched with the architectures

Reply:
While it's true that vedantmgoyal2009's PRs sometimes mismatch URLs and Architectures, here I believe the situation is that the unique installers for `x86` and `arm` were replaced with the `x64`.